snippets/numpy/__init__.py

```diff
@@ -0,0 +1,5 @@
+
+from . import core
+from .core import *
+from . import lib
+from .lib import *
```

Comment:
> @wired8 Thanks, this is really fantastic. I would have only one general comment. You have a lot of helper functions, e.g., `asarray`, `isreal` etc., and it would be great to have a top-level module or something like that, where one can easily include these functions without having to go through the complete `snippets` directory. Do you think you could add that?
> 
> Later, some of these functions could be implemented in C, but that is for the longer term.

Hi @v923z thank you.  When I originally wrote the helper functions the thought was to keep them in 1 file (module), but as they are essentially implementations of Numpy functions I decided it would be better to keep them in the same files as their equivalent Numpy source for easy reference.   I don't believe this is a major problem as the set_module decorator function brings them all under the Numpy namespace.

Reply:
You are right. Let's leave it at that!